ichnaea/scripts/load_cell_data.py

```diff
@@ -0,0 +1,55 @@
+#!/usr/bin/env python
+"""
+Load data from a downloaded public export (full or differential)
```

Comment:
Short answer: I wrote this docstring first, and the docs last, and changed my mind in between. It should change to match the docs, but maybe have some more details.

I haven't tried to download or import a full diff. The code should work, but it would probably require changes to ensure the database could grow to a very large size, that the memory limits were correct, etc. On Docker for Mac, this requires changing from the default settings, and I wouldn't know those limits until I tried it. 

There is also the chance that the full import contains data that I'm not expecting. For example, the cell station validator has an upper range for the radius, and some of the differential exports exceed this range.

In my experience, days can be lost hitting some unexpected data, a full disk, or needing to reboot, and then needing to start over again. I decided to skip importing a full export for now, but I'm not aware of any reason it couldn't be imported someday.

Reply:
I'm updating the docstring to:

```
"""
Load data from a downloaded public export

Download from https://location.services.mozilla.com/downloads

This has been tested with a differential cell export (~400kB compressed)
but not a full export (~370,000kB compressed)
"""
```

However, I'm reluctant to add this without trying it once, so I'm testing an import of the full export.

Reply:
It fails after 106,000 stations, when processing the second batch of cell areas ("Processed 2000 station areas").  The last few lines of the traceback:

```
  File "/app/ichnaea/data/tasks.py", line 141, in update_cellarea
    area.CellAreaUpdater(self)()
  File "/app/ichnaea/data/area.py", line 28, in __call__
    self.update_area(session, areaid)
  File "/app/ichnaea/data/area.py", line 148, in update_area
    region = self.region(ctr_lat, ctr_lon, mcc, cells)
  File "/app/ichnaea/data/area.py", line 47, in region
    [k for k, v in grouped_regions.items() if v == max_count]
TypeError: '<' not supported between instances of 'str' and 'NoneType'
```

I think "Import a full public export" should be a new task, and not block this PR.